scss/mixins/_banner.scss

```diff
@@ -0,0 +1,10 @@
+@mixin bsBanner($file, $suffix:"") {
+  /*!
+   * Bootstrap #{$file} v5.2.0-beta1 (https://getbootstrap.com/)
```

Comment:
In order to avoid having an extra-space when the file var is empty you can do the following:
```diff
diff --git a/scss/mixins/_banner.scss b/scss/mixins/_banner.scss
index ae73f8baa..87adb7bc4 100644
--- a/scss/mixins/_banner.scss
+++ b/scss/mixins/_banner.scss
@@ -1,6 +1,6 @@
-@mixin bsBanner($file, $suffix:"") {
+@mixin bsBanner($file:"", $suffix:"") {
   /*!
-   * Bootstrap #{$file} v5.2.0-beta1 (https://getbootstrap.com/)
+   * Bootstrap #{if($file!='',$file + ' ',$file)}v5.2.0-beta1 (https://getbootstrap.com/)
    * Copyright 2011-2022 The Bootstrap Authors
    * Copyright 2011-2022 Twitter, Inc.
    * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
```
It means also calling it `@include bsBanner();` in _scss/bootstrap.scss_